src/components/header/Header.vue

```diff
@@ -274,6 +299,10 @@ export default defineComponent({
       user-select: none;
     }
 
```

Comment:
Similarly to the hover style, I think we also need an active style for the selected mode, now that all options are displayed in a tab-like manner. Maybe you can play around with the looks of it. My tip would be that `--qalendar-blue` or `--qalendar-blue-transparent` looks OK here.